maplibre/src/lib.rs

```diff
@@ -108,10 +134,24 @@ where
     SM: ScheduleMethod,
     HC: HTTPClient,
 {
+    /// Runs the map application without a maximum number of frames per second defined.
+    ///
+    /// # Panics
+    ///
+    /// * Panics if the operating system is unable to starts the worker threads.
```

Comment:
```suggestion
    /// * Panics if the platform is unable to start the worker threads.
```

Reply:
Is that really the case? All unwraps in the code currently should basically go away soon. So I think we can avoid document panics.